specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -2591,6 +2591,10 @@
         "maxConcurrentConnections": {
           "type": "object",
           "description": "The maximum concurrent connection count for the sink data store. Type: integer (or Expression with resultType integer)."
+        },
+        "tableOption": {
```

Comment:
New writable properties should go in a new api-version. what is the impact of this value being set via portal, resource retrieved via an existing SDK which ignores this value since it doesn't understand it. Now the user sends a PUT on the existing resource from the SDK but the request body doesn't contain this. what happens to the value of this property on the server.

Reply:
- What is the impact of this value being set via portal：
The table in the target side will auto create for copy job. 

- What happens to the value of this property on the server if request body doesn't contain this:
the default value is none. 

This is a very common qestion for new release feature and backword compatiable 
 @hvermis help to explain the ADF SDK behavior.

Reply:
The SDK will ignore it if it doesn't know about it. We keep updating our SDKs frequently to keep and ask our customers to get the latest version. We also align new features in our UI with our SDK releases, and our customers know that if they use both, they need to get the latest SDK.
This actually a downside of Autorest - it should have kept original properties instead of discarding them, but that is a different topic.

Reply:
@hvermis - you do realize that this is contrary to the Microsoft Rest API guidelines which are publicly available? Please check it here - https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#123-definition-of-a-breaking-change
paragraph 2 sentence 2 in the above section.

Reply:
@ravbhatnagar Yes, we are aware. However those guidelines are not scalable - we always need to add new properties to our APIs and models, and adding new API version every time is not feasible.

Reply:
@ravbhatnagar  Do you have any other question?  Can you help to approve this PR?